src/extent.c

```diff
@@ -2162,24 +2164,58 @@ extent_merge_default_impl(void *addr_a, void *addr_b) {
 	return false;
 }
 
-#ifdef JEMALLOC_MAPS_COALESCE
+static bool
```

Comment:
Let's add a function-level comment? I.e.

"Returns true if the given extents can't be merged because of their head bit settings."

Reply:
Added.